pulp_container/app/registry_api.py

```diff
@@ -693,7 +693,7 @@ def put(self, request, path, pk=None):
 
             upload.delete()
 
-            job = dispatch(
+            dispatched_task = dispatch(
```

Comment:
It would now be fine to just call it `task` already. But that is not a necessary requirement.